op-node/node/client.go

```diff
@@ -135,16 +141,48 @@ type L1EndpointConfig struct {
 	// L1RPCKind identifies the RPC provider kind that serves the RPC,
 	// to inform the optimal usage of the RPC for transaction receipts fetching.
 	L1RPCKind sources.RPCProviderKind
+
+	// RateLimit specifies a self-imposed rate-limit on L1 requests. 0 is no rate-limit.
```

Comment:
L1 requests in the L2 config?

Reply:
It's part of the `L1EndpointConfig`, I think the github diff folding makes it hard to read. It's not part of the l2 engine or l2 sync configs.